benchmarks/user/benchmark.py

```diff
@@ -333,7 +334,7 @@ def param_string(self, params):
             devito_params['to'] = params['time_order']
             devito_params['dse'] = params['dse']
             devito_params['dle'] = params['dle']
-            devito_params['at'] = params['autotune']
+            devito_params['at'] = 'aggressive'
```

Comment:
Same note as above.

Reply:
why?

Reply:
what does `params['autotune']` contain? I don't remember

Reply:
It results to either `True` or `False`, making it non-compatible with `aggressive`,  `basic` or related """OFF"" version...

Reply:
maybe then you should use this? `configuration['autotuning'].level`

see [here](https://github.com/opesci/devito/blob/master/benchmarks/user/benchmark.py#L181)

don't know, but hardcoding "aggressive" doesn't look like a fix

Reply:
Tried that on Friday but did not work, will try again.